### PR TITLE
Tooltip update

### PR DESCRIPTION
**Pull request by emensch**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist
- [ ] All applicable changes have been documented
- [ ] Any `TO DO` items (or similar) have been entered as GitHub issues and the link to that issue has been included in a comment

### CODE REVIEW

File: packages/c2pa-wc/src/components/Popover/Popover.ts [hunk 1]
```diff
@@ -154,7 +153,7 @@ export class Popover extends LitElement {
           color: var(--cai-popover-color, #222222);
           transition-property: transform, visibility, opacity;
           border-radius: var(--cai-popover-border-radius, 6px);
-          border-width: var(--cai-popover-border-radius, 1px);
+          border-width: var(--cai-popover-border-width, 1px);
```
Comment (emensch) [inline packages/c2pa-wc/src/components/Popover/Popover.ts:156]: nothing to see here, please move along

File: packages/c2pa-wc/src/components/Popover/Popover.ts [hunk 2]
```diff
@@ -119,16 +119,16 @@ export class Popover extends LitElement {
     if (this.shift) {
       middleware.push(shift(this.shift));
     }
+    if (this.autoPlacement) {
+      middleware.push(autoPlacement(this.autoPlacement));
+    }
```
Comment (dkozma) [inline packages/c2pa-wc/src/components/Popover/Popover.ts:124]: Ahh, autoPlacement _before_ arrow.

File: packages/c2pa-wc/src/components/Popover/Popover.ts (hunk 1 above)
Comment (dkozma) [reply packages/c2pa-wc/src/components/Popover/Popover.ts:156]: 🚶🏼

File: packages/c2pa-wc/src/components/Popover/Popover.ts [hunk 3]
```diff
@@ -247,12 +243,56 @@ export class Popover extends LitElement {
...
+      bottom: '',
+      right: '',
+    };
+
+    // Split to turn left-end, left-start, etc into left
+    switch (placement.split('-')[0]) {
+      case 'bottom':
+        return {
+          ...base,
+          top: '-4px',
```
Comment (dkozma) [inline packages/c2pa-wc/src/components/Popover/Popover.ts]: Nit: Should we make `-4px` a const?

**Review by dkozma — APPROVED** [review]
Optional nit, but LGTM!

**Pull request merged**